src/apps/concepts/__test__/pages/ViewConceptsPage.test.tsx

```diff
@@ -55,20 +55,15 @@ function renderUI(props: Partial<viewConceptsPageProps> = {}) {
 }
 
 describe('ViewConceptsPage', () => {
-    it('should contain header', () => {
-        const {getByTestId} = renderUI(baseProps);
-        expect(getByTestId("header")).not.toBeNull();
-    });
 
-    it('should contain conceptsTable', () => {
+    it('should contain all the main component in page', () => {
```

Comment:
Instead of combining the tests, keep the individual tests and use a beforeEach block to declare const,

beforeEach(() => {
  // setup a DOM element as a render target
 const {getByTestId} = renderUI(baseProps);
});

Reply:
Yes @rakesh-1010 thought the same but it's actually replacement of snapshot test so we thought it would be more meaningful like this